src/MicrosoftTeams.ts

```diff
@@ -830,6 +838,29 @@ namespace microsoftTeams {
     sendMessageRequest(parentWindow, "openFilePreview", params);
   }
 
+  export interface ShowNotificationParameters {
+    message: string;
+    isDownloadComplete: boolean;
```

Comment:
I assume you will change this to an enum, right?

Reply:
yes